### DISCUSSION TITLE
Sub-Layouts are ready to test!

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#706] [Show and tell]
Sub-Layouts are ready to test. You can save sections as Sub-Layouts and reuse them in other templates. Let's download v3-branch and test-it today.
https://github.com/templaza/astroid-framework/tree/v3

![Screen Shot 2024-06-19 at 15 08 11](https://github.com/templaza/astroid-framework/assets/3204103/c7482fdd-18bf-4bf9-ae1c-c35843e97f1c)
![Screen Shot 2024-06-19 at 15 08 35](https://github.com/templaza/astroid-framework/assets/3204103/394b4187-1f55-4f63-b366-9d81e4a66226)
![Screen Shot 2024-06-19 at 15 08 49](https://github.com/templaza/astroid-framework/assets/3204103/f4105e06-4298-4945-9a07-6d451243c652)

Thanks & Best Regards,
Sonny

**Comment by Chacapamac** [comment]
This seem pretty cool, I need to update my local server to continue testing Astroid, Getting there slowly 🙁

**Reply by paurray** [reply]
@Chacapamac 
Are you still on MAMP? (they actually brought out 6.9!!!)
this really helped me:
https://apple.stackexchange.com/questions/471470/how-can-i-force-an-install-of-an-intel-binary-on-an-apple-silicon-machine-in-th

**Reply by Chacapamac** [reply]
I turn to XAMPP but it slow down my  interest to continue working on my websites tremendously. I go to MAMP right now.
Big Thanks @paurray Paul —> Did you test it?

**Reply by paurray** [reply]
@Chacapamac
hi Patrick
Tried XAMPP. Could not get my head around XAMPP!
Maybe it is just not great on a Mac. No clue.

Moved to Herd as they are very pro active with respect to getting MAMP customers onboard = importing sites from MAMP path!?!! Have not tried importing anything from MAMP though.

https://herd.laravel.com/

Half a year ago I also needed to install DBning.

https://dbngin.com/

This tutorial really helped me hook everything up:

https://www.youtube.com/watch?v=Rbt4Ldt0KU8&t=0s

UPDATE 01: 

I can not get MAMP to work :-(
Not sure if this is correct but it looks as if MAMP Pro 6.9.1 has PHP 8.1.13 and 8.2 at least it is there in the default PHP options. But I think the real problem is that the whole MySQL server is running 5.7.39. So I think MAMP remains a dead end! Sorry if I got your hopes up.

UPDATE 02:

I think if you watch this and the other You Tube video I posted above both from Susan B. You should be able to get up and running again. **Obviously back up every thing before trying any of this.** Hope this helps somehow.

https://www.youtube.com/watch?v=2hDQp6M42hg&t=1s

**Comment by Banuk1616** [comment]
I just wanted to take a look and test it, but I don't know exactly what to do here:

 https://github.com/templaza/astroid-framework/tree/v3

do I need to download?


Do I have to use the green button at the top right to download the code as a zip or the button at the top left to download "V3"?


Kind regards

**Reply by sonvnn** [reply]
You can download it direct here https://github.com/templaza/astroid-framework/archive/refs/heads/v3.zip

Thank you & Best regards,
Sonny

**Comment by Banuk1616** [comment]
I just did a quick test and noticed the following.


When I want to create a new section and the "Select Grid Layout" window opens, I can no longer cancel the process. The cross at the top right for dragging does not work.

When I have created a new section in the Sub Layout and press the Save button, the message Sub-layout is not saved appears at the bottom right. However, the sub-layout has been saved.

If I click on the Save & Close button and enter the title and description and then click on Save & Close again, a message also appears at the bottom right that the sub-layout has not been saved. But it has been saved.

My local environment:

Bearsampp Wampp
PHP 8.3.7
MariaDB 11.3.2

Translated with DeepL.com (free version)

**Reply by sonvnn** [reply]
Can you make a video describing this issue? Please clear the cache when reinstalling the Framework

Best Regards,
Sonny

**Reply by Banuk1616** [reply]
I'm a bit short of time at the moment, I'll see if I can do it next week.
